### ISSUE TITLE
Brackets 1.14.1 not installing under Ubuntu 20.04.1

### ISSUE THREAD

**Issue by kathe** [adobe/brackets#15297]
### Unable to install latest Brackets .deb under Ubuntu LTS

Attempting to install the Brackets 1.14.1 .deb under Ubuntu 20.04.1 LTS throws up a lot of unmet dependency errors and none of the dependency packages are available for installation anymore.

### Versions

Ubuntu 20.04.1 LTS (with all updates and upgrades installed).
Brackets 1.14.1 using the .deb file.

**Comment by irdroid3** [comment]
I followed [Jacobs][1] answer in - [askubuntu](https://askubuntu.com/questions/1061431/how-to-have-both-libcurl3-and-libcurl4-installed-at-same-time#)
I am using the deb file ([Brackets.Release.1.14.1.64-bit.deb][2]) on Ubuntu 20.04; but I had a problem at step 2.  I assume some change to the way things work...
Here are the steps that worked for me :

 1. Open terminal in temporary folder. Use :

    `dpkg-deb -x /path/to/deb/file .`

    `dpkg-deb -e Brackets.Release.1.14.1.64-bit.deb`

 2. Edit the control file : ./DEBIAN/control

 3. Change line with dependency from `libcurl3` to `libcurl3 | libcurl4`

 4. I deleted the original deb file - I am not sure I needed to?

 5. Run : `dpkg-deb -b . Bracket_CURL4.deb`

 6. Launch file browser and double click on the created deb file to launch the install.

 7. All done, **Brackets** program is installed.

  [1]: https://askubuntu.com/users/814675/jacob
  [2]: https://github.com/adobe/brackets/releases

**Comment by kathe** [comment]
Your instructions worked like a charm.
Thanks a million.  :-)
Just one more question; post installation via the file browser, can I delete the "usr", "opt" and "DEBIAN" directories from my home directory?

**Comment by irdroid3** [comment]
> Just one more question; post installation via the file browser, can I delete the "usr", "opt" and "DEBIAN" directories from my home directory?
Yes, those directories and their content are used to rebuild the .deb file.
You do not need to keep the .deb file either (unless you want a backup, and/or to install it elsewhere or if you rebuild your systrem again, etc...)
